lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -125,8 +174,8 @@ def declare_suspicious_replicas_bad(once: bool = False, younger_than: int = 5, n
 
 def run_once(heartbeat_handler: Any, younger_than: int, nattempts: int, vos: Optional[list[str]], limit_suspicious_files_on_rse: int, json_file_name: str, active_mode: int, **_kwargs) -> bool:
```

Comment:
```suggestion
def run_once(heartbeat_handler: Any, younger_than: int, nattempts: int, vos: Optional[Sequence[str]], limit_suspicious_files_on_rse: int, json_file_name: str, active_mode: int, **_kwargs) -> bool:
```

Imported from `collections.abc`